lib/disc.rb

```diff
@@ -110,10 +119,14 @@ def disc_options
       end
 
       def queue
-        disc_options.fetch(:queue, 'default')
+        @queue || Disc.default_queue
```

Comment:
From a user perspective I would find it a bit confusing that I have to set the queue via a plain class accessor, but the other options with the disc/disc_options/disque_options dsl, but I agree that it would make the code clearer.